ballista/rust/scheduler/src/lib.rs

```diff
@@ -722,33 +765,38 @@ impl SchedulerGrpc for SchedulerServer {
     ) -> std::result::Result<Response<ExecuteQueryResult>, tonic::Status> {
         if let ExecuteQueryParams {
             query: Some(query),
-            settings,
+            settings: _,
         } = request.into_inner()
         {
             // parse config
-            let mut config_builder = BallistaConfig::builder();
-            for kv_pair in &settings {
-                config_builder = config_builder.set(&kv_pair.key, &kv_pair.value);
-            }
-            let config = config_builder.build().map_err(|e| {
-                let msg = format!("Could not parse configs: {}", e);
-                error!("{}", msg);
-                tonic::Status::internal(msg)
-            })?;
+            // let mut config_builder = BallistaConfig::builder();
+            // for kv_pair in &settings {
+            //     config_builder = config_builder.set(&kv_pair.key, &kv_pair.value);
+            // }
+            // let config = config_builder.build().map_err(|e| {
+            //     let msg = format!("Could not parse configs: {}", e);
+            //     error!("{}", msg);
+            //     tonic::Status::internal(msg)
+            // })?;
 
             let plan = match query {
-                Query::LogicalPlan(logical_plan) => {
+                Query::LogicalPlan(message) => {
                     // parse protobuf
-                    (&logical_plan).try_into().map_err(|e| {
-                        let msg = format!("Could not parse logical plan protobuf: {}", e);
-                        error!("{}", msg);
-                        tonic::Status::internal(msg)
-                    })?
+                    // let ctx = create_datafusion_context(&config);
```

Comment:
left over comment?

Reply:
Why this logic was removed ?

Reply:
This was an oversight. I did not think these settings were actually used anywhere but was not accounting for the SQL case. Discussing fixes on https://github.com/apache/arrow-datafusion/issues/1848